vueui/src/components/view/devices/aicard.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div>
+    <template v-if="has_ai">
+      <div class="item">
+        <div class="itemLabel">
+          Hardware Integrity:
+        </div>
+        <div class="itemContent">
+          {{hardware_integrity}}%
+        </div>
+        <div class="itemLabel">
```

Comment:
Please use vui-group and vui-group-item for replacement of item[Label, Content] classes used in NanoUi